atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweeperImpl.java

```diff
@@ -197,7 +199,9 @@ long grabLocksAndRun(SweepLocks locks) throws InterruptedException {
             } else {
                 SweepBatchConfig lastBatchConfig = getAdjustedBatchConfig();
                 log.error("The background sweep job failed unexpectedly with batch config {}"
```

Comment:
We should discuss this (and I realise not the scope of your PR) but `ERROR` feels excessive here. I'd prefer `WARN` if >1 and `ERROR` only if =1. (Don't block on this comment!)